docs/fsharp/language-reference/symbol-and-operator-reference/index.md

```diff
@@ -70,9 +67,12 @@ The following table describes symbols used in the F# language and provides a bri
 |`<>?`|[Nullable Operators](nullable-operators.md)|<ul><li>Computes the "not equal" operation when the right side is a nullable type.<br /></li></ul>|
 |`<=`|[Arithmetic Operators](arithmetic-operators.md)|<ul><li>Returns `true` if the left side is less than or equal to the right side; otherwise, returns `false`.<br /></li></ul>|
 |`<=?`|[Nullable Operators](nullable-operators.md)|<ul><li>Computes the "less than or equal to" operation when the right side is a nullable type.<br /></li></ul>|
+|<code>&#124;></code>|[Functions](../functions/index.md)|<ul><li>Passes the result of the left side to the function on the right side (forward pipe operator).<br /></li></ul>|
+|<code>&#124;&#124;></code>|[&#40; &#124;&#124;&#62; &#41;&#60;'T1,'T2,'U&#62; Function](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html#(%20||%3E%20))|<ul><li>Passes the tuple of two arguments on the left side to the function on the right side.<br /></li></ul>|
```

Comment:
The problem here is `||` in the URL. Markdown parsers aren't able to detect these as part of the link and they see it as ending the current table cell.